src/Common/src/Interop/Windows/user32/Interop.MessageBox.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class User32
+    {
+        internal const int MB_ICONHAND = 0x00000010;
```

Comment:
Is this the complete set of MB_ values or would we expect any other library to pass other values in?  If this is the full set, we could make these an enum and then change the type of 'type' to be that enum, strongly-typing it.  Otherwise this is fine.